### PR TITLE
Bug 1373420 - Font for User Story is Courier instead of Fira Sans

### PR DESCRIPTION
**Pull request by kyoshino**
https://bugzilla.mozilla.org/show_bug.cgi?id=1373420

### CODE REVIEW

File: skins/standard/global.css [hunk 1]
```diff
@@ -35,7 +35,8 @@
     }
     /* monospace is much smaller than Verdana by default, so we make it a bit bigger. */
     pre, code, kbd {
-        font-size: medium;
+        font-family: "Droid Sans Mono", Menlo, Monaco, "Courier New", Courier, monospace;
+        font-size: small;
```
Comment (floatingatoll) [inline skins/standard/global.css]: Why is the font size of the User Story being reduced as part of changing the font, when one of the listed permitted fonts is the original font?

File: skins/standard/global.css [hunk 2]
```diff
@@ -35,7 +35,8 @@
     }
     /* monospace is much smaller than Verdana by default, so we make it a bit bigger. */
     pre, code, kbd {
-        font-size: medium;
+        font-family: "Droid Sans Mono", Menlo, Monaco, "Courier New", Courier, monospace;
```
Comment (floatingatoll) [inline skins/standard/global.css]: Does this set of changes bring the User Story font-size and font-family into alignment with the other instances of our use of Droid?

File: extensions/BugModal/web/bug_modal.css [hunk 3]
```diff
@@ -310,6 +310,7 @@ input[type="number"] {
 
 #user-story {
     margin: 0;
+    padding: 4px;
```
Comment (floatingatoll) [inline extensions/BugModal/web/bug_modal.css]: There are undocumented padding changes included in this font-fixing commit, that might be reversed undesirably if the font change goes wrong for any reason. Please update the commit message or split into a separate commit so that we're not secretly bringing along more than just 'Font'.

File: skins/standard/global.css [hunk 4]
```diff
@@ -304,7 +306,6 @@ div#docslinks {
 
 /* tbody.file pre is for the Diff view of attachments. */
 .bz_comment_text, .uneditable_textarea, tbody.file pre {
-     font-family: monospace;
```
Comment (floatingatoll) [inline skins/standard/global.css]: Is this change to bz_comment_text necessary to bring Fira Sans to the User Story field?

File: skins/standard/global.css [hunk 5]
```diff
@@ -1566,10 +1566,6 @@ body {
     font-size: small;
 }
 
-pre {
```
Comment (floatingatoll) [inline skins/standard/global.css]: This seems like it could have much wider effects and isn't obviously linked to the font size change. Have we vetted all `<pre>` uses to ensure that new alignment issues are not introduced by this change?

**Comment by kyoshino** [comment]
This PR has stalled for a while. I've found another `<textarea>` using Courier, and will redesign the entire site based on the new design system which includes different font choices.
